test/testinput/bufr_adpsfc.yaml

```diff
@@ -0,0 +1,168 @@
+# (C) Copyright 2020 NOAA/NWS/NCEP/EMC
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
```

Comment:
We are under LGPL license.
Not everything copy/paste is correct 😉

Reply:
I'll do you one better and get rid of all of it.

Reply:
Y E S